packages/router-generator/src/config.ts

```diff
@@ -13,6 +13,8 @@ export const configSchema = z.object({
   disableTypes: z.boolean().optional().default(false),
   addExtensions: z.boolean().optional().default(false),
   disableLogging: z.boolean().optional().default(false),
+  prependToRouteTreeFile: z.array(z.string()).optional().default([]),
```

Comment:
Let me know if you've got a better name for this configuration field.

Reply:
how about `routeTreeFileHeader` ?